src/interfaces.ts

```diff
@@ -36,6 +44,7 @@ export interface IFormProps<T = any> {
         inputs: any;
         isValid?: boolean;
         validationErrors: { [key: string]: string };
+        validationWarnings: { [key: string]: string };
```

Comment:
should it be something like `[key in keyof T]: string`?

Reply:
Yep 👍🏻